### ISSUE TITLE
Add support for Ripple panic button

### ISSUE THREAD

**Issue by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [Kunzisoft/KeePassDX#475]
Currently KeePass DX is not compatible with Ripple, it would be really nice and helpful for both projects if support would be added. KP DX would have an extra layer of security and Ripple would have one more app that has support, and this would incentivize Ripple's devs to continue with their work.

It would be even greater it you could add a setting to delete the entries, not just to hide the app when triggered. A lot of apps have support for it but only enable this option, which is kind of lame since it can have a bigger potential.

Thanks!

**Comment by J-Jamet** [comment]
Linked to https://github.com/Kunzisoft/KeePassDX/issues/129

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
> Linked to #129

Great!  Are you one of the devs?

**Comment by J-Jamet** [comment]
Yes, but I have not yet studied the impact it will have on the application. I already have to study how it works.
https://guardianproject.info/code/panickit/
https://github.com/guardianproject/ripple
